internal/memory/types.go

```diff
@@ -5,8 +5,8 @@ import "github.com/pkg/errors"
 type classType uint32
 
 const (
-	MegaByte = uint64(1024 * 1024)
-	GigaByte = 1024 * MegaByte
+	MegaByte uint64 = 1024 * 1024
+	GigaByte uint64 = 1024 * MegaByte
```

Comment:
Technically this should be [Gibibyte](https://simple.wikipedia.org/wiki/Gibibyte). Maybe we can fix that too?

Reply:
Sure, maybe we just shorten these to MiB and GiB then as well. Thoughts? cc @anmaxvl

Reply:
I like that idea even more :)

Reply:
sounds good to me 😄

Reply:
Changin in a follow up commit that we can squash at the end

Reply:
Updated